### PR TITLE
fixed timing issue in MaintainOperatorValueService

### PR DESCRIPTION
**Pull request by samt1803**
## Summary

Please provide a summary of the changes and a motivation if applicable.

## Changes

Provide a bullet list of individual changes. Leave this section out if change
set is small and obvious from summary.

## Limitations and future improvements

Provide a bullet list or description of known omissions or limitations of the
solution and/or any ideas for future improvement. Leave section ouf if
not applicable.

## Checklist before requesting a review

- [ ] Is this a breaking change? If it is, be clear in summary.
- [ ] Read through code myself one more time.
- [ ] Make sure any and all `TODO` comments left behind are meant to be left in.
- [ ] Has reasonable passing test coverage?
- [ ] Updated changelog if applicable.
- [ ] Updated documentation if applicable.

### CODE REVIEW

**Review by samigabor — APPROVED** [review]

File: packages/broker/.vscode/launch.json [hunk 1]
```diff
@@ -21,6 +21,26 @@
           "DEBUG": "*",
           "STREAMR_DOCKER_DEV_HOST": "localhost"
         }
+      },
+      {
```
Comment (harbu) [inline packages/broker/.vscode/launch.json:25]: Intentionally committed?

File: packages/broker/test/integration/plugins/operator/MaintainOperatorValueService.test.ts [hunk 2]
```diff
@@ -87,20 +87,22 @@ describe("MaintainOperatorValueService", () => {
...
     test.each([parseEther("0.001"), parseEther("0.0005")])("updates the sponsorships to stay over the threshold", async (penaltyFraction) => {
-        const poolValue = STAKE_AMOUNT * 2
-        const threshold = penaltyFraction.mul(poolValue).toBigInt()
         const maintainOperatorValueService = new MaintainOperatorValueService(operatorConfig, penaltyFraction.toBigInt())
 
         const totalValueInSponsorshipsBefore = await operatorContract.totalValueInSponsorshipsWei()
 
         // wait for sponsorships to accumulate earnings so approximate values differ enough form the real values
-        await wait(3000)
+        await wait(6000)
```
Comment (harbu) [inline packages/broker/test/integration/plugins/operator/MaintainOperatorValueService.test.ts]: Is it possible to use `waitForCondition` instead here in conjunction with queries to a JSON RPC for smart contract state?

File: packages/broker/.vscode/launch.json (hunk 1 above)
Comment (samt1803) [reply packages/broker/.vscode/launch.json:25]: yes, it is useful to all vscode users to run things in debug

File: packages/broker/test/integration/plugins/operator/MaintainOperatorValueService.test.ts (hunk 2 above)
Comment (samt1803) [reply packages/broker/test/integration/plugins/operator/MaintainOperatorValueService.test.ts]: yes, indeed, that is much better!

**Comment by samt1803** [comment]
There is a new commit on the branch, but but this PR is not picking it up for 3 hours now. Github status is all green, no apparent issues.
Ill try converting to draft and back.

**Review by harbu — APPROVED** [review]

**Pull request merged**